python/cudf/cudf/utils/dtypes.py

```diff
@@ -65,11 +66,25 @@
 }
 OTHER_TYPES = {"bool", "category", "str"}
 ALL_TYPES = NUMERIC_TYPES | DATETIME_TYPES | TIMEDELTA_TYPES | OTHER_TYPES
+NEW_NUMERIC_TYPES = {
+    "Int8",
+    "Int16",
+    "Int32",
+    "Int64",
+    "UInt8",
+    "UInt16",
+    "UInt32",
+    "UInt64",
+    "Float32",
+    "Float64",
+}
```

Comment:
Do we need these separate from the classic dtype strings? They resolve to the same types.